### ISSUE TITLE
md-tabs: Additional spaces in directive causing invalid style

### ISSUE THREAD

**Issue by Londovir** [angular/material#5593]
As of at least the current HEAD build, there appears to be an extra set of spaces in the `md-tabs` directive which is causing invalid styles to be applied in IE11 and Chrome:

![image](https://cloud.githubusercontent.com/assets/11982575/10998714/2bb84f06-8465-11e5-8a10-9d46debd3440.png)

I've tracked it down to Line 137 in tabsDirective.js: https://github.com/angular/material/blob/master/src/components/tabs/js/tabsDirective.js#L137

Once the space before the mustaches and the space after the mustaches are removed, it seems for me in IE11 and Chrome that all goes back to working well.

**Comment by Londovir** [comment]
Actually, now removing the space has not completely fixed the issue. In Chrome, the tabs are coming up with a `max-width: 0px` value initially on page load, and as soon as I roll my mouse cursor over a button I have in an `md-sidenav` component, the tabs apparently somehow reevaluate their styling and the correct value is being inserted. (If I click in the partially visible 1st tab, and the ripple begins firing, it also recalculates the width styles and the tabs appear). In IE11 it appears to now be working correctly from what I can see.

FWIW, I am doing my tab panel inside of a `ui-view`, so that could be playing a part somehow in this. Can't nail it down enough for replication in a CodePen yet, though. And if I navigate the view away in Chrome, and come back to it, I can't cause it to retrigger either. Odd.

**Comment by Splaktar** [comment]
I am seeing a mismatch of brackets, I don't think spaces is the issue. I.e.`{{ }`

**Comment by Londovir** [comment]
@Splaktar I think that was how IE11 was somehow interpretting the space in the directive template. If you look in the line I noted from the `md-tabs` file, it's definitely got the right number of closing mustaches. Removing the spaces from the front and back solved IE11's problem, but exposed the other one.

Looking over other tickets, it looks like this is hitting #4237. It sounds exactly like the same root problem, and appears to be related to `md-tabs` being contained in various types of other directives/containers. In that issue, it looks to be dialogs and/or sidenavs, but I'm hitting it now in a div with a ui-view attached.

I've also just now tested and show the issue in Firefox as well.

**Comment by Londovir** [comment]
Just tested against the most recent commit (v1.0.0-rc3-master-1ae16cb), and now even in IE11 I'm seeing odd behavior with the tabs which I wasn't getting the other day:

![tabsissue](https://cloud.githubusercontent.com/assets/11982575/11038713/5f7f9816-86d2-11e5-93ef-baceccd06761.gif)

The containing panel is not being resized in any manner. All I have are the tabs (which should show 3, with the first one "Schools" including a count of entries found in parentheses), a text input, and a table underneath that which is being filtered via the text input you see typing going into. If I create a filter that begins to restrict the number of entries, which in turn will cause the tab label to be revised and shortened, eventually it's enough that the final tab just simply disappears from view. If I reduce the filter to bring the first tab length back to being longer, the tab on the end will reappear.

BTW, I am back to seeing the odd, not-evaluated binding in IE11's developer panel again, like I first reported in the screenshot at the top. I've removed the additional spaces, but it's still doing this odd binding error in IE11. At this point, I cannot debug any further because I can't determine why IE11 is getting an un-eval'd value in that style; other bindings appear to be getting their calculated values inserted into the markup, but not this style entry.
